workspaces/adoption-insights/package.json

```
@@ -13,7 +13,7 @@
     "tsc:full": "tsc --skipLibCheck true --incremental false",
     "build:backend": "yarn workspace backend build",
     "build:all": "backstage-cli repo build --all",
-    "build:api-reports": "yarn build:api-reports:only --tsc",
+    "build:api-reports": "yarn build:api-reports:only",
```

Revert this change, as you can directly use `yarn build:api-reports:only` to build reports without typescript checks.
```suggestion
    "build:api-reports": "yarn build:api-reports:only --tsc",
```